src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt

```diff
@@ -58,6 +59,37 @@ class UserController(
   ) = userService.findUserByUsername(username)
     ?: throw NotFoundException("Account for username $username not found")
 
+  @GetMapping("/users/me")
+  @Operation(
+    summary = "My User details.",
+    description = "Find my user details."
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK",
+        content = [
+          io.swagger.v3.oas.annotations.media.Content(
```

Comment:
import is already there : io.swagger.v3.oas.annotations.media

Reply:
doh!